internal/component/pyroscope/scrape/scrape.go

```diff
@@ -181,16 +182,18 @@ func (cfg *ProfilingConfig) SetToDefault() {
 }
 
 type ProfilingTarget struct {
-	Enabled bool   `alloy:"enabled,attr,optional"`
-	Path    string `alloy:"path,attr,optional"`
-	Delta   bool   `alloy:"delta,attr,optional"`
+	Enabled bool      `alloy:"enabled,attr,optional"`
+	Path    string    `alloy:"path,attr,optional"`
+	Delta   bool      `alloy:"delta,attr,optional"`
+	Scaler  []float64 `alloy:"scaler,attr,optional"`
```

Comment:
maybe rename these to `local_scaler` just in case there is an remote scaler in the [future](https://github.com/grafana/pyroscope/issues/3979)? 

Or maybe keep it scaler, and later add a bool flag `scaler_local=true`